documentation/API-onboarding.md

```diff
@@ -43,7 +43,7 @@ In parallel, the API proposal is formally registered in CAMARA by adding it to t
 
 Upon receiving the API proposal and notification from the WG chair, the TSC studies the proposal and votes it at the TSC meeting.
 
-- No-Go! Objecting companies shall provide justifications why. The API proposal is rejected and will not be included in any API Sub Project.
+- No-Go! The API proposal is rejected and will not be included in any API Sub Project, and shall either (1) be removed from the API Backlog, with objecting companies to provide justifications why, or (2) returned to the API Backlog, with the TSC including in the 'justications' section the  clarifications required/gaps to be addressed/corrections required in order for the API to be re-submitted by the API supporters.
```

Comment:
Following suggestions discussed off-line, it is proposed to replace "No-Go!" (sounds a bit harsh, i.e. "No way you will make it into CAMARA with your proposal") with "Not accepted". 

I also suggests a rewording for option description: "Not approved: The API proposal is rejected, and thus will not be included in any API Sub-Project. The TSC will need to inform of this decision to the API backlog WG, and clarify next steps: either (1) remove API proposal from backlog, with objecting companies providing justifications why; or (2) ask for changes (e.g., clarifications, corrections, gaps to be addressed) required in order for the API to be re-submitted by the API supporters.

Reply:
@jordonezlucena @Kevsy The proposed text looks good to me, it's clearer how the decision is communicated (it would otherwise overload the table).

Reply:
Latest commit changes: 
Go! -> Accepted
No-Go! -> Not Accepted